src/mixins/nft.js

```diff
@@ -271,6 +274,35 @@ export default {
         ? this.listingInfo.price
         : this.purchaseInfo.price;
     },
+    isCollectExpiryTimeNear() {
+      const { collectExpiryAt } = this.purchaseInfo;
+      if (!collectExpiryAt) return false;
+
+      const timeLeft = collectExpiryAt - Date.now();
+      const threshold = 4 * 7 * 24 * 60 * 60 * 1000; // 4 weeks
+      return timeLeft > 0 && timeLeft < threshold;
+    },
+    collectExpiryTime() {
+      const { collectExpiryAt } = this.purchaseInfo;
+      if (!collectExpiryAt) return '';
+
+      const timeLeft = collectExpiryAt - Date.now();
+      if (timeLeft <= 0) return '';
+
+      const dateTime = DateTime.fromMillis(collectExpiryAt);
+      if (this.isCollectExpiryTimeNear) {
+        const duration = dateTime
+          .toRelative({ locale: this.$i18n.locale })
+          .replace('in ', '') // en
+          .replace('後', ''); // zh
```

Comment:
Just stick with english to extract the number/hour, and format it with `$t` instead? otherwise we need to add one `.replace` line for each locale here, which is unexpected when we are adding new locale

Reply:
Though luxon is `the evolution of Moment.js`, it does not support our use case, I replaced it with day.js.
https://momentjs.com/docs/#/-project-status/recommendations/
https://github.com/moment/luxon/issues/460